Bruin Bite/View Controllers/ProfileViewController.swift

```diff
@@ -9,12 +9,17 @@
 import UIKit
 import Foundation
 
-class ProfileViewController: UIViewController, ReadDelegate, AlertPresentable {
+class ProfileViewController: UIViewController, ReadDelegate, AlertPresentable, LoginAlertPresentable {
 
     @IBOutlet weak var profilePic: UIImageView!
     @IBOutlet weak var userName: UILabel!
     @IBOutlet weak var yearMajor: UILabel!
     @IBOutlet var ShortBio: UITextView!
+
+    override func viewDidAppear(_ animated: Bool) {
```

Comment:
Looks good, but out of curiosity, why did you choose `viewDidAppear` and not `viewDidLoad`? I would definitely test this out to make sure there aren't infinite alert loops if you keep coming back to the screen.

Reply:
I tested it out and there are not infinite alert loops if you keep coming back to the screen. If you think about it, in a tabBarController, when you switch to a different view controller it loads and displays and it stays. So, if I were to put it in viewDidLoad, my alert would trigger the first time, but it will not trigger all the other times I revisit the view controller.